.github/workflows/label_prs.yml

```
@@ -1,13 +1,15 @@
 name: 'Pull Request Labeler'
 
 on:
-  pull_request_target:
+  pull_request_target: [open, assigned, path]
```

actionlint (1.7.12)

[error] "pull_request_target" section is sequence node but mapping node is expected

(syntax-check)